geomesa-tools/bin/geomesa

```diff
@@ -7,85 +7,12 @@
 # http://www.opensource.org/licenses/apache2.0.php.
 #
 
-setGeoHome ()
-{
-    SOURCE="${BASH_SOURCE[0]}"
-    # resolve $SOURCE until the file is no longer a symlink
-    while [ -h "${SOURCE}" ]; do
-        bin="$( cd -P "$( dirname "${SOURCE}" )" && pwd )"
-        SOURCE="$(readlink "${SOURCE}")"
-        # if $SOURCE was a relative symlink, we need to resolve it relative to the path where
-        # the symlink file was located
-        [[ "${SOURCE}" != /* ]] && SOURCE="${bin}/${SOURCE}"
-    done
-    bin="$( cd -P "$( dirname "${SOURCE}" )" && cd ../ && pwd )"
-    export GEOMESA_HOME="$bin"
-    export PATH=${GEOMESA_HOME}/bin:$PATH
-    echo >&2 "Warning: GEOMESA_HOME is not set, using $GEOMESA_HOME"
-}
-
-GEOMESA_OPTS="-Duser.timezone=UTC -DEPSG-HSQL.directory=/tmp/$(whoami)"
-GEOMESA_OPTS="${GEOMESA_OPTS} -Djava.awt.headless=true"
-GEOMESA_DEBUG_OPTS="-Xmx8192m -XX:MaxPermSize=512m -XX:-UseGCOverheadLimit -Xdebug -Xnoagent -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=9898"
-GEOMESA_CP=""
-JAVA_LIBRARY_PATH=""
-
-
-
-# Define GEOMESA_HOME and update the PATH if necessary.
-if [ -z "$GEOMESA_HOME" ]; then
-setGeoHome
-else
-    echo >&2 "Using GEOMESA_HOME = $GEOMESA_HOME"
-    if [[ $1 = configure ]]; then
-        read -p "Do you want to reset this? Y\n " -n 1 -r
-        if [[  $REPLY =~ ^[Yy]$ ]]; then
-            echo >&2 ""
-            setGeoHome
-        fi
-        echo >&2 ""
-    fi
-fi
-
-# GEOMESA paths, GEOMESA_LIB should live inside GEOMESA_HOME, but can be pointed elsewhere
-if [ -z "$GEOMESA_LIB" ]; then
-    GEOMESA_LIB=${GEOMESA_HOME}/lib
-elif [[ $1 = configure ]]; then
-    message="Warning: GEOMESA_LIB was already set, probably by a prior configuration."
-    message="${message}\n The current value is ${GEOMESA_LIB}."
-    echo >&2 ""
-    echo -e >&2 "$message"
-    echo >&2 ""
-    read -p "Do you want to reset this to ${GEOMESA_HOME}/lib? Y\n " -n 1 -r
-    if [[  $REPLY =~ ^[Yy]$ ]]; then
-        GEOMESA_LIB=${GEOMESA_HOME}/lib
-        echo >&2 ""
-        echo "Now set to ${GEOMESA_LIB}"
-    fi
-    echo >&2 ""
-fi
-
-if [ -z "$GEOMESA_CONF_DIR" ]; then
-    GEOMESA_CONF_DIR=${GEOMESA_HOME}/conf
-elif [[ $1 = configure ]]; then
-    message="Warning: GEOMESA_CONF_DIR was already set, probably by a prior configuration."
-    message="${message}\n The current value is ${GEOMESA_CONF_DIR}."
-    echo >&2 ""
-    echo -e >&2 "$message"
-    echo >&2 ""
-    read -p "Do you want to reset this to ${GEOMESA_HOME}/conf? Y\n " -n 1 -r
-    if [[  $REPLY =~ ^[Yy]$ ]]; then
-        GEOMESA_CONF_DIR=${GEOMESA_HOME}/conf
-        echo >&2 ""
-        echo "Now set to ${GEOMESA_CONF_DIR}"
-    fi
-    echo >&2 ""
-fi
+# Set environment variables in bin/geomesa-env.sh
 
-GEOMESA_OPTS="${GEOMESA_OPTS} -Dlog4j.configuration=file://${GEOMESA_CONF_DIR}/log4j.properties"
+# Load common functions and setup
+. ./geomesa-common.sh
```

Comment:
Actually, I'm going to take a look at this more tomorrow, special characters in the path may break this as well.